python/example_code/lookoutvision/csv_to_manifest.py

```diff
@@ -0,0 +1,202 @@
+
+            metadata = {}
+            metadata['confidence'] = 1
+            metadata['job-name'] = "labeling-job/anomaly-classification"
+            metadata['class-name'] = row[1]
+            metadata['human-annotated'] = "yes"
+            metadata['creation-date'] = datetime.now(timezone.utc).strftime('%Y-%m-%dT%H:%M:%S.%f')
+            metadata['type'] = "groundtruth/image-classification"
+
+            json_line['anomaly-label-metadata'] = metadata
+
+            output_file.write(json.dumps(json_line))
+            output_file.write('\n')
+            image_count += 1
+
+    logger.info("Finished creating manifest file %s.\n"
+                "Images: %s\nAnomalous: %s",
+                manifest_file,
+                image_count,
+                anomalous_count)
+    return image_count, anomalous_count
+
+
+def add_arguments(parser):
+    """
+    Adds command line arguments to the parser.
+    :param parser: The command line parser.
+    """
+
+    parser.add_argument(
+        "csv_file", help="The CSV file that you want to process."
+    )
+
+    parser.add_argument(
+        "--s3_path", help="The S3 bucket and folder path for the images."
+        " If not supplied, column 1 is assumed to include the S3 path.", required=False
+    )
+
+
+def main():
+
+    logging.basicConfig(level=logging.INFO,
+                        format="%(levelname)s: %(message)s")
+
+    try:
+
+        # get command line arguments
+        parser = argparse.ArgumentParser(usage=argparse.SUPPRESS)
+        add_arguments(parser)
+        args = parser.parse_args()
+        s3_path = args.s3_path
+        if s3_path is None:
+            s3_path = ""
+
+        csv_file = args.csv_file
+        csv_file_no_extension = os.path.splitext(csv_file)[0]
+        manifest_file = csv_file_no_extension + '.manifest'
+
+        # Create manifest file if there are no duplicate images.
+        if check_errors(csv_file):
+            print(f"Issues found. Use {csv_file_no_extension}_errors.csv "\
+                "to view duplicates and errors.")
+            print(f"{csv_file}_deduplicated.csv contains the first"\
+                "occurence of a duplicate.\n"
+                  "Update as necessary with the correct information.")
+            print(f"Re-run the script with {csv_file_no_extension}_deduplicated.csv")
+        else:
+            print('No duplicates found. Creating manifest file')
+
+            image_count, anomalous_count = create_manifest_file(csv_file, manifest_file, s3_path)
+
+            print(f"Finished creating manifest file: {manifest_file} \n")
+
+            normal_count = image_count-anomalous_count
+            print(f"Images processed: {image_count}")
+            print(f"Normal: {normal_count}")
+            print(f"Anomalous: {anomalous_count}")
+
+    except FileNotFoundError as err:
+        logger.exception("File not found.:%s", err)
+        print(f"File not found: {err}. Check your input CSV file")
```

Comment:
print(f"File not found: {err}. Check your input CSV file.")

Reply:
Fixed.